<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="face_register_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 247, 48, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>얼굴 등록</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="payment_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 247, 48, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>물품 결제</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>face-register.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>payment.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
